core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java

```diff
@@ -52,14 +53,18 @@ public class MetricsUtil {
   private static List<Tag> commonTags;
 
   public static void initializeMetrics(final AccumuloConfiguration conf, final String appName,
-      final HostAndPort address) throws Exception {
+      final HostAndPort address) throws ClassNotFoundException, InstantiationException,
+      IllegalAccessException, IllegalArgumentException, InvocationTargetException,
+      NoSuchMethodException, SecurityException {
     initializeMetrics(conf.getBoolean(Property.GENERAL_MICROMETER_ENABLED),
         conf.getBoolean(Property.GENERAL_MICROMETER_JVM_METRICS_ENABLED),
         conf.get(Property.GENERAL_MICROMETER_FACTORY), appName, address);
   }
 
   private static void initializeMetrics(boolean enabled, boolean jvmMetricsEnabled,
-      String factoryClass, String appName, HostAndPort address) throws Exception {
+      String factoryClass, String appName, HostAndPort address) throws ClassNotFoundException,
+      InstantiationException, IllegalAccessException, IllegalArgumentException,
+      InvocationTargetException, NoSuchMethodException, SecurityException {
```

Comment:
Both the throws here and the catches don't need to be so verbose with all these. ReflectiveOperationException would be a good choice to replace most of these.